tests/DependencyInjection/RoutingDebugReportTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+declare(strict_types=1);
+
+namespace League\Tactician\Bundle\Tests\DependencyInjection;
+
+use League\Tactician\Bundle\DependencyInjection\HandlerMapping\Routing;
+use League\Tactician\Bundle\DependencyInjection\RoutingDebugReport;
+use League\Tactician\Bundle\Tests\Fake\FakeCommand;
+use League\Tactician\Bundle\Tests\Fake\OtherFakeCommand;
+use PHPUnit\Framework\TestCase;
+
+class RoutingDebugReportTest extends TestCase
+{
+    public function testItShouldBuildValidReport()
+    {
+        // GIVEN
+        $buses = [
+            'default',
+            'foo',
+            'bar'
+        ];
+
+        $routing = new Routing($buses);
+
+        $routing->routeToAllBuses(FakeCommand::class, 'fake.handler');
+        $routing->routeToBus('foo', OtherFakeCommand::class, 'other_fake.handler');
+
+        $mappings = [];
+        foreach ($buses as $bus) {
+            $mappings[$bus] = $routing->commandToServiceMapping($bus);
+        }
+
+        // WHEN
+        $report = new RoutingDebugReport($mappings);
+
+        // THEN
+        $reportArray = $report->toArray();
+        $this->assertCount(3, $reportArray);
+        $this->assertArrayHasKey('default', $reportArray);
+        $this->assertArrayHasKey('foo', $reportArray);
+        $this->assertArrayHasKey('bar', $reportArray);
+
+        $this->assertEquals(
+            [FakeCommand::class => 'fake.handler', OtherFakeCommand::class => 'other_fake.handler'],
+            $reportArray['foo']
+        );
+
+        $this->assertEquals(
+            [FakeCommand::class => 'fake.handler'],
+            $reportArray['default']
+        );
+
+        $this->assertEquals(
+            [FakeCommand::class => 'fake.handler'],
+            $reportArray['bar']
+        );
+    }
```

Comment:
You can simplify this test a lot. Since we're only testing the RoutingDebugReport, you can just pass it a premade array, rather than creating a Routing object, etc. You can also test the entire array in a assert statement:
~~~php
$this->assertEquals(
    [
       'foo' => ['bar', 'blat']
    ],
    $myArray
);
~~~

That's totally fine, especially since this object is so tiny and minimal right now, there's very little to test. Kudos for the effort though!